docs/modules/sql/pages/partition-pruning.adoc

```diff
@@ -0,0 +1,63 @@
+= Partition pruning of SQL queries
+:description: The partition pruning optimization technique reduces latency for faster IMap SQL queries.
+:page-beta: false
+
+Partition pruning is an IMap-specific query optimization technique that automatically prunes members that don't contain
+the required data on their partitions, and optimizes the count of Jet scan processors on required members. It works:
+
+* by default, for primitive `__key` types (default attribute strategy).
+* for all key types if you configure the `AttributePartitioningStrategy` in `MapConfig`.
+
+NOTE: The partition pruning may be used *only* when querying an IMap mapping if the partitioning strategy attribute is enabled.
+
+NOTE: The query coordinator member is always involved in the query execution, even if it doesn't contain any
+required data.
+
+== Example
+
```

Comment:
can we have some intro text, something like "The following example shows the configuration updates for......." 

and include information on the filename/location requirements of the config file as well?

Reply:
As there are three examples, we need to add context for each

Reply:
> and include information on the filename/location requirements of the config file as well?

We don't have such for partition pruning. :)